### ISSUE TITLE
Adding all pandas reference  links to the documentations

### ISSUE THREAD

**Issue by sephib** [dask/dask#2589]
Hi
I thought of adding links to the pandas references in the documentation as a PR.
For example  in the section of [read_csv](http://dask.pydata.org/en/latest/dataframe-api.html#storage-and-conversion) add the link to [pandas.read_csv](https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html) in the `**kwargs` section.

Please let me know if this is wanted or are there other documentation amendments that are wanted.

**Comment by mrocklin** [comment]
Linking to other documentation seems fine to me.  What mechanism would you
use to do this?  I would be cautious about linking directly to Pandas docs
in case they move in the future.

**Comment by sephib** [comment]
I was planning to link using the `stable` word in the url  
i.e.  https://pandas.pydata.org/pandas-docs/stable/generated/pandas.read_csv.html  
Do you have a different suggestion?

**Comment by mrocklin** [comment]
@TomAugspurger are these likely to stick around?

**Comment by TomAugspurger** [comment]
Yeah that should be fine. Typically we'll redirect links for e.g. a renamed
method.

**Comment by sephib** [comment]
Hi,
I'm new to the `sphinx-doc`.
In which file do I add the link to ? can you maybe point out for an example the link to the `read_csv` (as suggested in the beginning of this thread?

**Comment by TomAugspurger** [comment]
@sephib To do this automatically for all methods would require some custom sphinx code (I'm not that familiar with it either).

For specific functions, you should be able to use [intersphinx](http://www.sphinx-doc.org/en/stable/ext/intersphinx.html). You'll add it to the extensions [here](https://github.com/dask/dask/blob/20a189cc886f440f461bfc97e5fdf7d2b9cb8a75/docs/source/conf.py#L28) and  include pandas in the mapping. Then you would update the docstring with `:meth:pandas.read_csv` and it should be turned into a link. The pandas inventory is at http://pandas.pydata.org/pandas-docs/stable/objects.inv
